Create custom "issue templates" for the repository (#2037)
hashicorp/design-system: .github/ISSUE_TEMPLATE/file-issue-external-users.md

---
name: File an issue (for external users)
about: Submit a request about our components or documentation (for non HashiCorp employees)
title: 'External request: [your-title-here]'
labels: external-request
---

<!--- Notice: using GitHub to file issues is only for external users -->

> <!--- If you are an HashiCorp employee refer to our support page: https://helios.hashicorp.design/about/support -->

Author: See comment above.

## Description of the problem
<!--- Describe what issue you have.  -->
<!--- Please include a screenshot or gif if possible! -->

## Possible suggested solution
<!--- (Optional) Suggest how you would solve it -->

<!-- ================================

⚠️ Disclaimer ⚠️

> While we don't preclude requests from external users, please consider that we have limited capacity to support them, being focused on internal activities.
> We may not answer in a timely manner, or provide the solution you are asking for.

Reviewer: ```suggestion
We have limited capacity to support external requests. As such, we may not answer in a timely manner or be able to provide the solution you're asking for.
```
